### ISSUE TITLE
Standardise the "Chip"/Badge component

### ISSUE THREAD

**Issue by CarlosNZ** [msupply-foundation/open-msupply#4779]
## The suggestion

For vaccination card, we created a Badge component for vaccination status, as per [the UI designs](https://github.com/msupply-foundation/open-msupply/issues/4663):

![360298251-3a602ca5-9cf7-477d-8d69-675f5dcc79e8](https://github.com/user-attachments/assets/0dcb76bb-209d-4616-a286-576b429a6f05)

However, there is already a slightly different type of Badge used in Cold chain: 
![Screenshot 2024-09-12 at 12 36 31 PM](https://github.com/user-attachments/assets/a85d436f-e408-4026-9a60-7e9e2999ce99)

The one in cold chain uses a component that is within the "cold-chain" package, so presumably not intended for use outside that.

I think we should agree on a standard Badge component then re-factor to use the same one everywhere for UI consistency. The one I made for Vax card is intentionally generic (it takes a `statusMap` prop to specify the colors/labels based on the input), so would be easy to re-use.

### Why should we invest time in this?

UI consistency, reduce code duplication.

### Are there any risks associated with this change?

Not that I can think of.

### How much effort is required?

Probably just a few hours.

### Agreed Solution

TBD, but based on the [Vaccine card designs](https://github.com/msupply-foundation/open-msupply/issues/4663), something like that is probably where we're going.

**Comment by mariyamsupply** [comment]
Agree, suggest we use created for vaccination status **Badge component**
